support/Environments/frontera_gcc.sh

```diff
@@ -181,20 +182,23 @@ EOF
         tar xzf v6.10.2.tar.gz
         mv charm-6.10.2 charm
         cd $dep_dir/charm
-        ./build charm++ mpi-linux-x86_64 smp mpicxx --with-production -j6
-        ./build LIBS mpi-linux-x86_64 smp mpicxx --with-production -j6
+        MPICC=mpicc MPICXX=mpicxx \
+             ./build charm++ mpi-linux-x86_64 smp mpicxx --with-production -j6
```

Comment:
You can delete this line. The `LIBS` target includes the `charm++` target, see [docs](https://github.com/UIUC-PPL/charm#building-the-source)